cmake/ProjectWasm3.cmake

```diff
@@ -11,22 +11,24 @@ set(binary_dir ${prefix}/src/wasm3-build)
 set(include_dir ${source_dir})
 set(wasm3_library ${binary_dir}/source/${CMAKE_STATIC_LIBRARY_PREFIX}m3${CMAKE_STATIC_LIBRARY_SUFFIX})
 
-# Should use a release, like https://github.com/wasm3/wasm3/archive/v0.4.6.tar.gz, once changes are upstreamed.
+# Should use a release, like https://github.com/wasm3/wasm3/archive/v0.4.7.tar.gz, once changes are upstreamed.
```

Comment:
This is actually used.

Reply:
I guess it is not relevant since we have the patches here. Removed.